include/seqan3/utility/views/enforce_random_access.hpp

```diff
@@ -0,0 +1,372 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ * \brief Provides seqan3::views::enforce_random_access.
+ */
+
+#pragma once
+
+#include <seqan3/range/views/detail.hpp>
+#include <seqan3/range/detail/inherited_iterator_base.hpp>
+#include <seqan3/range/concept.hpp>
+#include <seqan3/std/iterator>
+#include <seqan3/std/ranges>
```

Comment:
Also the include order could be fixed:
```suggestion
#include <seqan3/std/iterator>
#include <seqan3/std/ranges>

#include <seqan3/range/concept.hpp>
#include <seqan3/range/detail/inherited_iterator_base.hpp>
#include <seqan3/range/views/detail.hpp>
```